src/containers/Transactions/index.js

```diff
@@ -33,12 +34,15 @@ ERROR_MESSAGES.default = {
 
 const getErrorMessage = error => ERROR_MESSAGES[error] || ERROR_MESSAGES.default;
 
-class Transaction extends Component {
-  componentDidMount() {
-    const { t, actions, match, data } = this.props;
+const Transaction = props => {
+  const { identifier = '', tab = 'simple' } = useParams();
+  const match = useRouteMatch();
+  const { t } = useTranslation();
+  const { actions, data, loading } = props;
```

Comment:
extreme nit - usually props destructuring happens as the first line in the component. just a pattern. does not block this pr